webrender/src/tiling.rs

```diff
@@ -818,6 +825,28 @@ enum AlphaBatchKind {
     BoxShadow,
 }
 
+bitflags! {
+    pub flags AlphaBatchKeyFlags: u8 {
+        const NEEDS_CLIPPING  = 0b00000001,
+        const AXIS_ALIGNED    = 0b00000010,
+        const COMPLEX         = 0b00000100,
```

Comment:
I think we can remove the COMPLEX flag altogether, since it's mutually exclusive with AXIS_ALIGNED and isn't actually used.